test/integration_tests/test_container/build/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY deps/01_nodoc /etc/dpkg/dpkg.cfg.d/01_nodoc
+
+RUN apt-get -y update && \
```

Comment:
also run `apt-get upgrade` here?

Reply:
apt-get upgrade is unusual in dockerfiles, because the assumption is that the base image is fairly up to date.